test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] I see that you added `test.md` to your `gitignore` file but still I can see it in your repo please fix that. You can try removing test.md from your [ranch](https://github.com/AceMannie/Hello-Microverse/tree/ranch) branch on your repo.